notification/twilio/twiml.go

```diff
@@ -133,8 +142,18 @@ func (t *twiMLResponse) sendResponse() {
 		xml.EscapeText(t.w, []byte(t.gatherURL))
 		io.WriteString(t.w, `">`+"\n")
 	}
+
+	// add optional voice and/or language options
+	voiceAndLanuage := ""
+	if t.voiceName != "" && t.voiceLanguage != "" {
+		// language is required with a voice
+		voiceAndLanuage = fmt.Sprintf(` voice="%s" language="%s"`, t.voiceName, t.voiceLanguage)
+	} else if t.voiceLanguage != "" {
+		// language can be set without a voice
+		voiceAndLanuage = fmt.Sprintf(` language="%s"`, t.voiceLanguage)
+	}
```

Comment:
nit: we can simplify this and avoid the `else` by just doing an `if` for each, the name-then-lang requirement is already enforced by config validation.

```go
var opts string
if voice {
  opts += ` voice="%s"`
}
if lang {
  opts += ` lang="%s"`
}
```

**However,** the bigger issue here is that we need to escape the values otherwise this is vulnerable to TwiML injection through the UI.

We could lift and use the escaping code from the XML lib [here](https://cs.opensource.google/go/go/+/refs/tags/go1.19:src/encoding/xml/xml.go;drc=8d57f4dcef5d69a0a3f807afaa9625018569010b;l=1961), but I'm thinking it might be a good sign that it's time to use the proper XML library to build this.

Reply:
Updated to use the xml parser.